bin/rose-config-edit

```diff
@@ -50,16 +50,21 @@
 #        Launch, ignoring any configuration.
 #     --no-metadata
 #        Launch with metadata switched off.
-#     --no-warn-version
-#        Suppress "could not find metadata for app/version, using app/HEAD"
-#        warnings.
+#     --no-warn
```

Comment:
Should say `--no-warn=WARNING-TYPE` so it is clear to user that the option expects an argument.